guides/v2.1/cloud/project/new-relic.md

```diff
@@ -34,26 +66,40 @@ runtime:
 
 ## Add New Relic APM to an environment
 
-The {{site.data.var.ece}} plans to support up to three instances of your New Relic APM license across all environments. We recommend adding a New Relic license to your Staging and Production environments, and you can add the license to one other environment of your choice.
+The {{site.data.var.ece}} Starter and Pro plans support up to three instances of your New Relic APM license across all environments. We recommend adding a New Relic license to your Staging and Production environments, and you can add the license to one other environment of your choice. 
+
```

Comment:
Should we emphasize that if you want to add to an Integration environment, you have to submit a support ticket to request installation of the New Relic APM agent on the integration environment?

Reply:
My understanding is the warning was added because the service key is **inherited** through the variables whenever an environment is created. If they want to add more licenses than allotted, they must purchase through New Relic. If, for some strange reason, that was not set up before they created Integration, and there is something else they must do to get New Relic to work in a PaaS environment, then maybe, but you will have to confirm that with someone familiar. By default, they can only run in three environments and the third one is their choice by adding the service key.